doc/source/dev_guide/testing.rst

```diff
@@ -0,0 +1,6 @@
+Testing Satpy-based applications
+================================
+
+The ``satpy.testing`` modules provides tools for writing tests of applications that use Satpy.
+
+- For faking reading, you can use :func:`~satpy.testing.fake_satpy_reading`
```

Comment:
This isn't a hard requirement/change for this to be merged, but this wording of "faking reading" seems not as descriptive as I'd like. On one hand I would expect the high-level description to be something like "To allow Scene creation and loading fake datasets without reading files", but that's kind of long. My main point being that "For faking reading" doesn't seem to fully describe what is possible, or rather, why you'd use this functionality.

Reply:
I like your wording, so I'll put it in here. The docstring of that function also has an example, so that part is quite clear I think.